### DISCUSSION TITLE
Check SSH keys

### DISCUSSION THREAD

**Discussion by jone** [ParetoSecurity/pareto-mac#79] [Ideas]
In our company, we'd like to make sure that all employees are using "good" ssh keys.
Our rule:
1. use a good encryption (e.g. ed25519)
2. use a passphrase

It would be great if pareto could check that and report when somebody has weak keys in their ´~/.ssh/` folder.

**Comment by zupo** [comment]
We have exactly this check already in the works. First version will check only for presence of passphrase, but I like the idea to also check the encryption. Besides `ed25519`, which encryption schemes would you consider "good"?

**Comment by jone** [comment]
For us, good is either an `ed25519` or an `rsa` with a strength of at least `4096`.
